systems/notification/docker-compose.yml

```diff
@@ -21,12 +21,12 @@ services:
    - DB_PASSWORD=Pass2020!
    - DB_USER=postgres
    - QUEUE_URI=amqp://guest:guest@${LOCAL_HOST_IP}:5672
-   - MAILER_HOST=${MAILER_HOST}
-   - MAILER_PORT=${MAILER_PORT}
-   - MAILER_PASSWORD=${MAILER_PASSWORD}
-   - MAILER_USERNAME=${MAILER_USERNAME}
+   - MAILERHOST=${MAILERHOST}
+   - MAILERPORT=${MAILERPORT}
+   - MAILERPASSWORD=${MAILERPASSWORD}
+   - MAILERUSERNAME=${MAILERUSERNAME}
```

Comment:
If you have configs with sub structures you need to add `_` in between. `i.e: MAILERHOST should be MAILER_HOST`